stdlib/3/builtins.pyi

```diff
@@ -278,15 +278,15 @@ class str(Sequence[str]):
     def strip(self, chars: Optional[str] = None) -> str: ...
     def swapcase(self) -> str: ...
     def title(self) -> str: ...
-    def translate(self, table: Dict[int, Any]) -> str: ...
+    def translate(self, table: Union[Mapping[int, Union[int, str, None]], Sequence[Union[int, str, None]]]) -> str: ...
     def upper(self) -> str: ...
     def zfill(self, width: int) -> str: ...
     @staticmethod
     @overload
-    def maketrans(x: Union[Dict[int, Any], Dict[str, Any]]) -> Dict[int, Any]: ...
+    def maketrans(x: Dict[Union[int, str], _T]) -> Dict[int, _T]: ...
```

Comment:
This is not good, since mypy now won't allow you to pass a `Dict[int, _T]` due to invariance. Can the `x` argument be a `Mapping` instead?

Reply:
It should be, actually.